pycti/connector/opencti_connector_helper.py

```diff
@@ -1556,6 +1556,8 @@ def send_stix2_bundle(self, bundle: str, **kwargs) -> list:
         :type entities_types: list, optional
         :param update: whether to updated data in the database, defaults to False
         :type update: bool, optional
+        :param bypass_split: @deprecated use to prevent splitting of the bundle. Not used anymore
```

Comment:
```suggestion
        :param bypass_split: @deprecated [>=6.3 & <6.6] use to prevent splitting of the bundle. Not used anymore
```

Consistent with our deprecation strategy, searching that pattern will allow us to clean up.
Shall I document this in the [breaking-changes update](https://github.com/OpenCTI-Platform/docs/pull/209/files) ?